package.json

```diff
@@ -707,6 +715,56 @@
           "when": "view =~ /^tekton(CustomTree|PipelineExplorer)View/ && viewItem == taskrun",
           "group": "c2"
         },
+        {
+					"command": "tekton.edit",
+          "when": "view =~ /^tekton(CustomTree|PipelineExplorer)View/ && viewItem == pipeline",
+					"group": "inline"
+        },
+        {
+					"command": "tekton.edit",
+          "when": "view =~ /^tekton(CustomTree|PipelineExplorer)View/ && viewItem == pipelinerun",
+					"group": "inline"
+        },
+        {
+					"command": "tekton.edit",
+          "when": "view =~ /^tekton(CustomTree|PipelineExplorer)View/ && viewItem == clustertask",
+					"group": "inline"
+        },
+        {
+					"command": "tekton.edit",
+          "when": "view =~ /^tekton(CustomTree|PipelineExplorer)View/ && viewItem == pipelineresource",
+					"group": "inline"
+        },
+        {
+					"command": "tekton.edit",
+          "when": "view =~ /^tekton(CustomTree|PipelineExplorer)View/ && viewItem == taskrun",
+					"group": "inline"
+        },
+        {
+					"command": "tekton.edit",
+          "when": "view =~ /^tekton(CustomTree|PipelineExplorer)View/ && viewItem == triggertemplates",
+					"group": "inline"
+        },
+        {
+					"command": "tekton.edit",
+          "when": "view =~ /^tekton(CustomTree|PipelineExplorer)View/ && viewItem == triggerbinding",
+					"group": "inline"
+        },
+        {
+					"command": "tekton.edit",
+          "when": "view =~ /^tekton(CustomTree|PipelineExplorer)View/ && viewItem == eventlistener",
+					"group": "inline"
+        },
+        {
+					"command": "tekton.edit",
+          "when": "view =~ /^tekton(CustomTree|PipelineExplorer)View/ && viewItem == conditions",
+					"group": "inline"
+        },
+        {
+					"command": "tekton.edit",
+          "when": "view =~ /^tekton(CustomTree|PipelineExplorer)View/ && viewItem == task",
+					"group": "inline"
+				},
```

Comment:
I think it would be better to use regexp in checking of `viewItem`, same as it was done for view.
We may have only one settings for `tekton.edit` command